lib/node_modules/@stdlib/blas/ext/base/gnansumkbn2/benchmark/benchmark.ndarray.js

```diff
@@ -38,17 +53,7 @@ var gnansumkbn2 = require( './../lib/ndarray.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = [];
-	for ( i = 0; i < len; i++ ) {
-		if ( randu() < 0.2 ) {
-			x.push( NaN );
-		} else {
-			x.push( ( randu()*10.0 ) - 20.0 );
-		}
-	}
+	var x = filledarrayBy( len, 'float64', rand );
```

Comment:
```suggestion
	var x = filledarrayBy( len, 'generic', rand );
```

Reply:
@headlessNode We should preserve the original dtype in this case, as this allows for comparing generic arrays against the `d*` and `s*` versions which operate against specific typed arrays.

This applies here and in other PRs.